### DISCUSSION TITLE
解决macOS下无法在vscode集成终端调试的问题

### DISCUSSION THREAD

**Discussion by Yu-Jiawei** [Yang-Xijie/yang-xijie.github.io#12] [Cpp in VSCode on macOS]
个人觉得使用外部中断还是有点略微麻烦
看到网上一些教程，可以使用CodeLLDB这个插件，然后将launch.json中的"type": "lldb", "externalConsole"设置为false，然后可以vscode内置的debugger里面实时调试了

**Comment by Levine66** [comment]
是的，这样的做法对M1系列的Mac同样适用

**Comment by gitony0101** [comment]
跟着你的视频和网站配置了我的tasks文件，谢谢啦！
